### ISSUE TITLE
tablet issue with spotify authentication

### ISSUE THREAD

**Issue by mczerski** [jaedb/Iris#200]
## Expected behavior
can authenticate to spotify

## Actual behavior
clicking on login button creates new tab in browser, in that tab I can login to spotify, but after loging, nothing happens in Iris

## Steps to replicate
1. open Iris on galaxy tabe in chrome
2. try to login to spotify

#### Operating systems affected
- [ ] Windows:
- [ ] Mac: 
- [ ] Linux: 
- [ ] iOS: 
- [ x] Andriod:
- [ ] Other:

#### Browsers affected
- [ x] Chrome: 
- [ x] Firefox: 
- [ ] Internet Explorer:
- [ ] Opera: 
- [ ] Other:

#### Screen sizes affected
- [ ] Huge (ie desktop)
- [ ] Large (ie laptop)
- [ x] Medium (ie slim laptop/tablet)
- [ ] Small (phone)

## Before you submit
- [ x] Yes, I have tested this issue thoroughly
- [ ] Yes, I have attached screenshots and/or screencast of the issue in action
- [ ] Yes, I have included any relevant debug data and error logs

**Comment by jaedb** [comment]
What version of Chrome browser and Android device are affected by this? I've just tested on Galaxy Tab 4 with Chrome 58 and all seems to function normally.

**Comment by mczerski** [comment]
chrome 60, android 4.4.4, tablet galaxy tab e 9.6 (sm-t560). I checked firefox and it is working correctly so the problem is only witch chrome.

**Comment by Hugomm** [comment]
Same problem on Chrome on another Android device. Tested on other browsers and working. It seems to be a Chrome issue.

**Comment by jaedb** [comment]
@Hugomm can you please confirm whether this issue still persists?

**Comment by Hugomm** [comment]
Not using Mopidy at the moment, maybe If I have time I will try it and feedback. Sorry @jaedb

**Comment by RensOliemans** [comment]
I have the same problem. When I attempt to log in, I get redirected to [:6680/iris/undefined?action=authorize&scope=playlist-modify-private%20playlist-modify-public%20playlist-read-private%20playlist-modify-private%20user-library-read%20user-library-modify%20user-follow-modify%20user-follow-read%20user-read-email%20user-top-read%20user-read-currently-playing%20user-read-playback-state%20playlist-read-collaborative%20ugc-image-upload](https://localhost:6680/iris/undefined?action=authorize&scope=playlist-modify-private%20playlist-modify-public%20playlist-read-private%20playlist-modify-private%20user-library-read%20user-library-modify%20user-follow-modify%20user-follow-read%20user-read-email%20user-top-read%20user-read-currently-playing%20user-read-playback-state%20playlist-read-collaborative%20ugc-image-upload)

Which is a 404.
However, when I enable SSL and Apply and Reload, it **does** work. However, when I have SSL enabled, I get _Mopidy-Spotify extension is not running - you will not be able to play any Spotify tracks_, which I otherwise do not get.

TL;DR, the authorization with Spotify doesn't work.
The browsers on which the error occurs are `Firefox 59.0.2` and `Chromium 65.0.3325.181`

Iris and Mopidy runs on Raspbian Stretch, Kernel version 4.14

**Comment by jaedb** [comment]
@RensOliemans I believe the issue you describe is covered in #282

**Comment by RensOliemans** [comment]
@jaedb Yes indeed, I'll delete this comment

**Comment by jaedb** [comment]
Closing due to inactivity. Please feel free to re-open if this is still a relevant concern.
